src/abi.h

```diff
@@ -249,6 +249,7 @@ struct netplan_net_definition {
         char* driver;
         char* mac;
         char* original_name;
+        char* pciid;
```

Comment:
todo: This has the potential of breaking ABI and we'd need to move the new field to the very end of the `netplan_net_definition` struct, e.g. called `match_pciid`.